Cubical/Foundations/Prelude.agda

```diff
@@ -160,6 +161,17 @@ compPathP {A = A} {x = x} {B = B} p q i =
                   (i = i1) → q j })
        (p i)
 
+
+compPathP' : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} {x y z : A} {x' : B x} {y' : B y} {z' : B z} {p : x ≡ y} {q : y ≡ z}
```

Comment:
I'm starting to this that all of these different version of compPath should be moved to Foundations.Path. Don't do it in this PR, but it might be good to do so that the Prelude is smaller